applications/salk-portal/src/models/ExperimentInstance.ts

```diff
@@ -0,0 +1,21 @@
+import {Experiment, ExperimentOwner, ExperimentOwnerGroups, ExperimentCollaborators, ExperimentPopulations, ExperimentTags} from '../apiclient/workspaces/api'
+
+export default class ExperimentInstance implements Experiment {
```

Comment:
@paularah I would just extend the interface and use it for the object creation.

when you do implements then you need to (re)write the whole interface when the generated api interface changes. Using extends will make the code more "agile"